images/linux/scripts/installers/packer.sh

```diff
@@ -5,7 +5,7 @@
 ################################################################################
 
 # Install Packer
-PACKER_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/packer | jq -r .current_version)
+PACKER_VERSION=$(curl -s https://api.releases.hashicorp.com/v1/releases/packer/latest | jq -r .version)
```

Comment:
Let's update the filter and get a link from json:
`'.builds[] | select((.arch=="amd64") and (.os=="linux")).url'`